RHEL/7/input/remediations/bash/accounts_tmout.sh

```diff
@@ -0,0 +1,7 @@
+# platform = Red Hat Enterprise Linux 7
+#. /usr/share/scap-security-guide/remediation_functions
+#popu#late var_#accounts_#tmout
+
+#echo "TMOUT=$var_accounts_tmout" >> /etc/profile
```

Comment:
The reason it is commented out is because var_accounts_tmout isn't a piped-through variable (presumably it will be one). I tried to figure out how to do it and got lost in the xml.